pkg/referenceutil/referenceutil_test.go

```diff
@@ -273,6 +273,10 @@ func TestReferenceUtil(t *testing.T) {
 			Tag:          "latest",
 			ExplicitTag:  "",
 		},
+		"oci-archive:///tmp/build/saved-image.tar": {
+			Protocol: "oci-archive",
```

Comment:
Protocol is not necessary here, since the test stops as it errors (and since your latest version does not set the protocol)